### PR TITLE
encfs: Update to 1.8.1

### PR DESCRIPTION
**Pull request by equalsraf**

### CODE REVIEW

File: common/shlibs [hunk 1]
```diff
@@ -2325,3 +2325,4 @@ libftdi1.so.2 libftdi1-1.2_1
 libax25.so.0 libax25-0.0.12rc4_1
 libax25io.so.0 libax25-0.0.12rc4_1
 libmill.so.12 libmill-1.4_1
+libencfs.so.6 encfs-1.8.1_1
```
Comment (Gottox) [inline common/shlibs]: No need for that. Ignore the warning.

Comment (equalsraf) [inline common/shlibs]: I've removed the line.

File: srcpkgs/encfs/template [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
 makedepends="fuse-devel libressl-devel librlog-devel boost-devel"
 short_desc="Encrypted filesystem in user-space"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-3"
 homepage="http://www.arg0.net/encfs"
-distfiles="http://encfs.googlecode.com/files/${pkgname}-${version}.tgz"
-checksum=282ef0f04f2dd7ba3527b45621fab485b7cc510c2ceee116600d0348dc2170a8
+distfiles="https://github.com/vgough/encfs/releases/download/v${version}/${pkgname}-${version}.tar.gz"
+checksum=8a0257ff500c14244ee99acdd472696966796e2a0931e4a132191f14a666d5d1
+# AC_TRY_COMPILE prevents cross compile
```
Comment (xtraeme) [inline srcpkgs/encfs/template]: I suppose you meant AC_TRY_RUN? AC_TRY_COMPILE works in cross compilation.

Comment (equalsraf) [inline srcpkgs/encfs/template]: Correct. Fixed the comment.

Comment (Gottox) [inline srcpkgs/encfs/template]: I still believe, that this call can be patched out. Anyway, this can be done later. Good to go. :+1:

**Pull request merged**